index.js

```diff
@@ -244,6 +248,10 @@ async function _render(view, options, hooks) {
       vnode.state = omit(component, COMPONENT_PROPS);
       vnode.attrs = component.attrs || view.attrs || {};
 
+      if (!component.view) {
```

Comment:
don't get this check, can you explain?

Reply:
Not sure how or why, but in my case there were times when `component.view` was empty, so it broke the rendering. This is what fixed it for me.